mmrazor/models/task_modules/estimators/counters/flops_params_counter.py

```diff
@@ -503,9 +511,13 @@ def get_counter_type(module):
 
 
 def is_supported_instance(module):
-    """Judge whether the module is in TASK_UTILS registry or not."""
+    """Judge whether the module can be countered or not."""
     if get_counter_type(module) in TASK_UTILS._module_dict.keys():
         return True
+    else:
+        for op in get_modules_list():
+            if issubclass(module.__class__.__base__, op):
```

Comment:
multiple inheritances should also be handled.

Reply:
Use `module.__class__.mro()` to handle multiple inheritances, done.